### PR TITLE
new Japanese traslation for senario

### PR DESCRIPTION
**Pull request by masuda2022**
They were not translated and shown in English partially, so I've made it.
 I am not familiar with GitHub, it may have problems with the way or format...

### CODE REVIEW

File: contribution/mobile/ja/procedural-names.json [hunk 1]
```diff
@@ -207,40 +199,43 @@
...
+				"ライトセイバー",
 				"拳銃",
 				"光線銃",
 				"ブラスター",
 				"銃",
 				"シューター",
-				"ザッパー"
+				"ザッパー",
+				"水鉄砲",
+				"輪ゴム鉄砲",
```
Comment (notmanz) [inline contribution/mobile/ja/procedural-names.json:238]: Please erase the comma (line 238) as it's the last variable in the array. Adding a comma without any final variable will result in an error.

File: contribution/mobile/ja/procedural-names.json [hunk 2]
```diff
@@ -250,6 +245,7 @@
 				"ライフル",
 				"キャノン砲",
 				"ザッパー"
+				"火炎放射器",
```
Comment (notmanz) [inline contribution/mobile/ja/procedural-names.json:248]: Add a comma (on line 247), and erase the comma (on line 248) as it's the last variable in the array.

Comment (notmanz) [reply contribution/mobile/ja/procedural-names.json]: Yes. Your changes were okay now.
You can see if the changes were correct or not by looking at the Check Pass below (green = good to go/pass the check).

As for the next question, you just need a bare minimum knowledge of code language (particularly JSON & Markdown) in order to do a PR change. Of course, it's normal to see a mistake / mistype whenever someone did a PR, as verifier/repo owner will tell you which part was the issue & will encourage you to fixes the issue (or in other cases, the verifier/repo owner will do these fixes & push it to the branch immediately). So.. nope. it didn't bother me (if that question were aim towards my previous comment) or other verifier/repo owner :)

Comment (masuda2022) [reply contribution/mobile/ja/procedural-names.json]: I'm relieved to hear that! This is my first time to join such community so I was worrying. Thank you all! :)

**Pull request merged**